risc0/zkvm/sdk/rust/build/README.md

```diff
@@ -1,6 +1,50 @@
 # risc0-build
 
-This crate contains helper functions to assist with building guest side code,
-and using it in host side code.
+Build RISC Zero zkVM guest code and provide handles to the host side
 
-[TODO: Examples]
+In order for the host to execute guest code in the [RISC Zero zkVM](risc0_zkvm), the host must be provided an ELF file with RISC-V source code and the corresponding [MethodID](risc0_zkvm::host::MethodId). This crate contains the functions needed to take zkVM guest code written in rust, build a corresponding ELF file and MethodID, and make the MethodID and a path to the ELF file available for the host to use.
```

Comment:
The ELF doesn't contain source code, but was compiled from it.

It's also not always the case that all the source code will be based on rust. It's possible to mix with other languages.